jdi-light/src/main/java/com/epam/jdi/light/asserts/generic/UIAssert.java

```diff
@@ -68,7 +68,7 @@ public A enabled() {
      */
     @JDIAction("Assert that '{name}' is disabled")
     public A disabled() {
-         jdiAssert(element.isEnabled() ? "enabled" : "disabled", is("disabled"));
+         jdiAssert(element.isDisabled() ? "disabled" : "enabled", is("disabled"));
```

Comment:
@RomanIovlev  we need your approve here

Reply:
I think we can apply that change without Roma, because there is no actual changes in business logic here

Reply:
Okay